### ISSUE TITLE
Pull is too long

### ISSUE THREAD

**Issue by g4s8** [zold-io/wts.zold.io#290]
README suggested to use `/pull` before each WTS access and says that it's fast operation:
> asks the server to pull your wallet from the network. This is a pretty fast and safe operation, you can do it every time before reading the wallet content, like finding transactions or checking the balance. If the wallet already exists on our server, there will be no pull from the network. If you really want to pull, no matter what, add force=true.

But actually it's quite expensive operations, it takes 15-20 seconds to pull and wait the job to complete. So I'd update the readme or make `/pull` faster.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@g4s8 are you sure it's slow? I just checked the code, it looks like it should be very fast, unless you use `force`.

**Comment by g4s8** [comment]
@yegor256 yes, it takes more than 10 seconds on each pull (even if my wallet is already pulled). I don't use force. This is what I'm doing to pull the wallet:
 - send `GET` - `/pull` to `https://wts.zold.io`
 - parse `X-Zold-Job` header
 - send `GET` - `/job?id=<job-id>` every second, until it returns `OK` or error
